com.github.rssguard.json

```diff
@@ -0,0 +1,34 @@
+{
+    "id": "com.github.rssguard",
+    "branch": "master",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.9",
+    "sdk": "org.kde.Sdk",
+    "command": "rssguard",
+    "rename-icon": "rssguard",
+    "finish-args": [ "--share=ipc",
+                     "--share=network",
+                     "--socket=x11",
+                     "--socket=wayland",
+                     "--filesystem=xdg-download",
+                     "--talk-name=org.kde.StatusNotifierWatcher",
+                     "--talk-name=org.freedesktop.Notifications",
+                     "--env=QT_QPA_PLATFORM=flatpak" ],
+    "modules": [
+        {
+            "name": "rssguard",
+            "buildsystem": "qmake",
+            "post-install": [
+                "install -d /app/share/icons/hicolor/512x512/apps",
+                "install -m644 /app/share/pixmaps/rssguard.png /app/share/icons/hicolor/512x512/apps/rssguard.png"
+            ],
+            "sources": [
+                {
+                    "type": "git",
```

Comment:
there's a source tar.gz of this [available as well](https://github.com/martinrotter/rssguard/releases), they're quite a lot faster and more efficient to build, any reason not to use it?